test/public/dataPasses/overviewPerLhcPeriod.test.js

```diff
@@ -191,14 +190,14 @@ module.exports = () => {
         // Expect a sorting preview to appear when hovering over a column header
         await page.waitForSelector('th#outputSize');
         await page.hover('th#outputSize');
-        await waitForTimeout(100);
+        await page.waitForSelector('#outputSize-sort-preview');
         const sortingPreviewIndicator = await page.$('#outputSize-sort-preview');
         expect(Boolean(sortingPreviewIndicator)).to.be.true;
```

Comment:
Here you simply remove the waitForTimeout but you don't wait for anything, so maybe the redraw is not done (tests may pass for a while, but it's only by luck)

Reply:
I replaces it with waiting for the preview selector to appear, is this not enough? But now after your feedback, `await page.$('#outputSize-sort-preview');` already does that so I think I can just remove the waitForSelector?

Reply:
Done, discussed on mattermost